### PR TITLE
chore: add metrics system prefix

### PR DESCRIPTION
**Pull request by googs1025**
For different components in the same project, metrics should be distinguished by different system prefixes.

### CODE REVIEW

**Review by fmuyassarov — COMMENTED** [review]
/lgtm

**Review by marquiz — CHANGES_REQUESTED** [review]
Thank you @googs1025 for the improvement.

Could you also update the docs (`docs/deployment/metrics.md`) and the example dashboard (`examples/grafana-dashboard.json`) to have them in sync?

**Comment by googs1025** [comment]
> Thank you @googs1025 for the improvement.
> 
> Could you also update the docs (`docs/deployment/metrics.md`) and the example dashboard (`examples/grafana-dashboard.json`) to have them in sync?

OK, I can do this weekend. 😄

**Comment by googs1025** [comment]
> Thank you @googs1025 for the improvement.
> 
> Could you also update the docs (`docs/deployment/metrics.md`) and the example dashboard (`examples/grafana-dashboard.json`) to have them in sync?

done

**Review by fmuyassarov — COMMENTED** [review]
/lgtm
ping @marquiz

File: docs/deployment/metrics.md [hunk 1]
```diff
@@ -13,24 +13,25 @@ By default NFD Master and Worker expose metrics on port 8081.
...
-| `nfd_gc_objects_deleted_total`                    | Counter   | Number of NodeFeature and NodeResourceTopology objects garbage collected. |
-| `nfd_gc_object_delete_failures_total`             | Counter   | Number of errors in deleting NodeFeature and NodeResourceTopology objects. |
+| Metric                          | Type      | Description                                                                |
+| ------------------------------- | --------- |----------------------------------------------------------------------------|
+| `nfd_master_build_info`         | Gauge     | Version from which nfd-master was built                                    |
+| `nfd_worker_build_info`         | Gauge     | Version from which nfd-worker was built                                    |
+| `nfd_gc_build_info`             | Gauge     | Version from which nfd-gc was built                                        |
+| `nfd_topology_updater_build_info` | Gauge     | Version from which nfd-topology-updater was built                          |
+| `nfd_master_node_update_requests_total` | Counter   | Number of node update requests received by the master over gRPC            |
```
Comment (marquiz) [inline docs/deployment/metrics.md:22]: small nit: could you keep the columns aligned

Comment (googs1025) [reply docs/deployment/metrics.md]: done

**Review by marquiz — APPROVED** [review]
Thank you @googs1025 for the update
/lgtm

**Pull request merged**